web/public/css/dashboard.less

```diff
@@ -62,11 +62,20 @@ label {
     padding: 46px 100px 0 100px;
 }
 
+
+.anchor-jump {
+  padding-top: 140px;
+  margin-top: -140px;
```

Comment:
So... I get what you're going for here... But it breaks things. Try add a youth under 13. ;)